server/github.go

```diff
@@ -166,6 +166,10 @@ func (s *Server) getMembers(ctx context.Context) (orgMembers []string, err error
 	var allUsers []*github.User
 	for {
 		users, r, err := s.GithubClient.Organizations.ListMembers(ctx, s.Config.Org, opts)
+		if _, ok := err.(*github.RateLimitError); ok {
+			s.sleepUntilRateLimitAboveTokenReserve()
```

Comment:
I'm 4/5 that is just buries the root issue. Roughly 4 requests are made there. They should always works.

Reply:
The rate limit api call isn't counted towards the rate limit.

Reply:
Are these changes this valuable?

Reply:
I'm 2/5 that these few requests here don't make a difference. Also, given that the code just keeps the old list of fetching the new fails, it's not worth re-trying.

Reply:
> What happened to the integrations tests?

The function for them was superflous. So I removed the file.